theano/sandbox/cuda/basic_ops.py

```diff
@@ -3329,20 +3329,42 @@ def c_code_cache_version(self):
         return ()
 
 
-class GpuFlatten(gof.HideC, tensor.Flatten, GpuOp):
+class GpuFlatten(gof.HideC, tensor.Reshape, GpuOp):
     """
     Implement Flatten on the gpu.
 
+    Note: The interface GpuFlatten is deprecated, you should use gpu_flatten
```

Comment:
.. note:: The interface GpuFlatten is deprecated, you should use gpu_flatten.

This will render more nicely on the website. :)